include/openenclave/enclave.h

```diff
@@ -724,6 +724,76 @@ oe_enclave_t* oe_get_enclave(void);
  */
 oe_result_t oe_random(void* data, size_t size);
 
+/**
+ * oe_generate_attestation_cert.
+ *
+ * This function generates a self-signed x509 certificate with an embedded
+ * quote from the underlying enclave.
+ *
+ * @param[in] subject_name a string contains contain an X.509 distinguished
+ * name (DN) for customizing the generated certificate. This name is also used
+ * as the issuer name because this is a self-signed certificate
+ * See RFC5280 (https://tools.ietf.org/html/rfc5280) specification for details.
+ * Example value "CN=Open Enclave SDK,O=OESDK TLS,C=US"
+ *
+ * @param[in] private_key a private key used to sign this certificate
```

Comment:
nit: some lines end in period (740, 742) and some don't (739, 741).  Be consistent throughout this file.

Reply:
thanks for spotting this. Fixed.

Reply:
Nit is not yet fixed.  See line 742